src/Symfony/Component/HttpFoundation/Cookie.php

```diff
@@ -296,20 +274,17 @@ public function getPath()
 
     /**
      * Creates a cookie copy with a new path on the server in which the cookie will be available on.
+     *
+     * @return static
      */
-    public function withPath(?string $path): self
+    public function withPath(string $path): self
```

Comment:
there is no need to make this one nullable as `null` is turned into a string internally